### ISSUE TITLE
Remove repeated accordion label and update styling 

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3448]
## Description
Content **SHOULD** clearly indicate what is related and what is distinct.

As a sighted user, when I open an accordion item in the sidenav, I might clearly understand how the content is related.

**[COGNITION]: Once opened, the "accordion" repeats the label, adds a horizontal rule separating related content**

This is confusing because it isn't clear why there is duplication, and the horizontal rule between would indicate they are separated. 

![pittsburgh-sidenav-open.png](https://images.zenhubusercontent.com/5d6685f1a77bcf04870bc81c/95bafe20-60e2-4cb8-a989-c2526bf90742)

## Type of Issue
Template

## Point of Contact

Jennifer
## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

1. Enter `https://staging.va.gov/pittsburgh-health-care/` in browser
2. Select the `^` to open an accordion
3. View the repeated link label plus the horizontal rule between them

## Tasks
[ ] REsolve bug causing duplication of accordion label 

## Acceptance Criteria
[ ] When user visits https://staging.va.gov/pittsburgh-health-care/` in browser and opens accordion, all child subpages (and their child subpages) are displayed in nesting style 
[ ] The selected page label is not duplicated.
[ ] Parent functionality is maintained (as it behaves on prod)
[ ] Horizontal rule separates content  
[ ] Content is clearly displayed in mobile view or when zoomed to 400%

## Possible Fixes (optional)

See tickets [#3441](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/3441) and [#3445](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/3445)

## WCAG Guidance 

* [WCAG 2.4.4 Link purpose in context](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)
* [WCAG 2.4.9 Navigation mechanisms, links](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html)]
The following issues are related to [WCAG Cognition](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/WCAG-Checklist.md).
* The information architecture of the site and design SHOULD be intuitive, allowing the user to easily
find, navigate, read, and interact with the content. 
* The design SHOULD eliminate or minimize visual distractions.
* The design SHOULD follow common conventions for design and user interaction, unless there is a
compelling (e.g. research-validated) reason to break from convention.

**Comment by davidconlon** [comment]
@jenstrickland  @rtwell
Can you confirm that this issue is/will be resolved through implementation of the revised design?

**Comment by rtwell** [comment]
yes, the new design / functionality removes the duplicity of the two parent level pages, and functions much like our nav does now: on-click not only takes the user to the parent page, but also (this is the new part) exposes all children and grandchildren

**Comment by jenstrickland** [comment]
The proposed new design, https://github.com/department-of-veterans-affairs/va.gov-team/issues/3413#issuecomment-590518022, removes the repetition, and should address this issue. @davidconlon

**Comment by mmiddaugh** [comment]
From @kelsonic 
Duplicate label will need to be removed: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/side-nav/components/NavItem.js#L39

**Comment by mmiddaugh** [comment]
@rtwell to add design specs

**Comment by mmiddaugh** [comment]
Merge to staging only

**Comment by brangi** [comment]
merged

**Comment by jenstrickland** [comment]
The horizontal rule is still there, separating the parent from the child. Can that rule be removed to meet the criteria of this ticket?

**Comment by mmiddaugh** [comment]
@jenstrickland the purpose of this particular ticket was the elimination of the duplicate label only. Styling will be addressed in a subsequent issue.

**Comment by jenstrickland** [comment]
This issue ticket included both items, so I wouldn't close this, yet.

**Comment by mmiddaugh** [comment]
I have added _- [ ] Horizontal rules do not separate parent from child items._ as acceptance criteria to [#7212]

**Comment by mmiddaugh** [comment]
@jenstrickland  this one is ready for you

**Comment by jenstrickland** [comment]
Validated. 🎉
